pkg/kubelet/kubelet.go

```diff
@@ -1223,10 +1229,17 @@ func (kl *Kubelet) computePodContainerChanges(pod *api.Pod, runningPod kubeconta
 	}, nil
 }
 
-func (kl *Kubelet) syncPod(pod *api.Pod, mirrorPod *api.Pod, runningPod kubecontainer.Pod) error {
+type syncPodSummary struct {
```

Comment:
WDYT: map of event types to counts? We wouldn't have to change this struct if more types are added.